charts/clamav/values.yaml

```
@@ -88,3 +88,6 @@ hpa:
   # average memory usage per pod (100Mi-1Gi)
   memory:
   requests:
+
+podDisruptionBudget: {}
```

historically, this has been done more like:
```suggestion
podDisruptionBudget:
  enabled: false
  maxUnavailable: 1
```

(https://github.com/helm/charts/search?q=podDisruptionBudget)